### DISCUSSION TITLE
Slight issue w initial setup

### DISCUSSION THREAD

**Discussion by swiperthefox93** [pierrelasse/YimStuff#10] [General]
Should this just merge w the YimMenu that comes up when I press insert or is there a separate key for it? Sorry if this is a dumb question I'm relatively new to all of this

**Comment by pierrelasse** [comment]
Hello, this is a script made for YimMenu that gets loaded by YimMenu. See quick installation in the README to install the script.

Then you just start gta, inject yimmenu and the script should be loaded.

You can find everything in the bottom tab. (damn i have to make tutorials and make the docs)

If you have any problems or help, just type here.
